### ISSUE TITLE
Certificate not loading

### ISSUE THREAD

**Issue by ni720** [haiwen/seadroid#587]
Hi,

I upgraded to Android7 (no factory reset) and after that I could not login to my seafile server anymore, because the certificate was not loading. I deleted all the known certificates, reinstalled seafile etc - did not help.
It works with other seafile servers (which have not been used before on this phone).
I suspect a certificate hickup somwhere, but I am not sure. I dont't know if a FactoryReset would help. Any Ideas?

Thx!

![tmp_26000-screenshot_20160915-1740391787854839](https://cloud.githubusercontent.com/assets/5901728/18580412/b9facf30-7bfb-11e6-962b-88718dbec121.png)

**Comment by fuelflo** [comment]
Just for info:

thats on Seadroid 2.1.6, Android 7.
Seafile server 6.0.3, valid let's encrypt certificate with A+ rating at ssllabs test.

Same server, android device with CM13 (Android 6.0.1) seadroid 2.1.6....no problems....as with the above mentioned android 7 device before the upgrade to android 7.

**Comment by ni720** [comment]
Thx for adding the infos @fuelflo.
After doing a factory reset, I can confirm, that it is an Android7 problem...

Any ideas, what could cause the problem?

**Comment by ni720** [comment]
I was able to identify the problem. It looks as if it is really an Android7 bug.
Having no idea what I'm actually doing, I disabled my nginxconfig line by line and identified the problematic line:
`ssl_ecdh_curve secp384r1;`
Android 7 cannot use this elliptic curve according to [this](http://stackoverflow.com/questions/39133437/sslhandshakeexception-handshake-failed-on-android-n-7-0)
`ssl_ecdh_curve prime256v1;` works.
For me this issue can be closed, as its not a Seadroid issue.
